worker/uniter/uniter.go

```diff
@@ -142,6 +143,15 @@ type Uniter struct {
 	// isRemoteUnit is true when the unit is remotely deployed.
 	isRemoteUnit bool
 
+	// containerNames will have a list of the sidecar containers created alongside this
```

Comment:
s/sidecar/workload ?

maybe name the attribute workloadContainerNames
(to match workloadEvents and workloadEvenChannel)